vowpalwabbit/core/src/reductions/ftrl.cc

```diff
@@ -32,6 +33,31 @@ using namespace VW::reductions;
 #define W_WE 4  // Wealth
 #define W_MG 5  // maximum gradient
 
+namespace VW
+{
+namespace reductions
+{
+namespace model_utils
+{
+size_t write_model_field(io_buf& io, ftrl& ftrl_data, const std::string& upstream_name, bool text)
```

Comment:
ftrl already has save resume, how is this different to what ftrl already does today?